README.md

```diff
@@ -112,6 +113,45 @@ export default ({store}) => {
 }
 ```
 
+#### Using cookies (universal client + server-side)
+Add `cookie` and `js-cookie`:
+
+`npm install --save cookie js-cookie`
+or `yarn add cookie js-cookie`
+
+```javascript
+...
+plugins: [{ src: '~/plugins/persistedState.js']
```

Comment:
Please add the comment `// nuxt.config.js` here so readers know where to add it.